Editor/Window/ManagedEC2Page.cs

```diff
@@ -0,0 +1,145 @@
+            _launchClientButton.SetEnabled(
+                _model.CurrentStackInfo.StackStatus is StackStatus.CreateComplete or StackStatus.UpdateComplete);
+
+            _deploymentScenariosInput.SetEnabled(_model.CanDeploy);
+            _fleetParamsInput.SetEnabled(_model.CanDeploy);
+            _ec2DeploymentStatusLabel.text = _model.CurrentStackInfo.StackStatus;
+        }
+
+        private void LocalizeText()
+        {
+            var l = new ElementLocalizer(_container);
+            var replacements = new Dictionary<string, string>()
+            {
+                { "GameName", Application.productName },
+                { "ScenarioType", GetScenarioType(l) }
+            };
+            l.SetElementText("ManagedEC2Title", Strings.ManagedEC2Title);
+            l.SetElementText("ManagedEC2Description", Strings.ManagedEC2Description);
+            l.SetElementText("ManagedEC2IntegrateTitle", Strings.ManagedEC2IntegrateTitle);
+            l.SetElementText("ManagedEC2IntegrateDescription", Strings.ManagedEC2IntegrateDescription);
+            l.SetElementText("ManagedEC2IntegrateLink", Strings.ManagedEC2IntegrateLink);
+            l.SetElementText("ManagedEC2ScenarioTitle", Strings.ManagedEC2ScenarioTitle);
+            l.SetElementText("ManagedEC2ParametersTitle", Strings.ManagedEC2ParametersTitle, replacements);
+            l.SetElementText("ManagedEC2DeployTitle", Strings.ManagedEC2DeployTitle, replacements);
+            l.SetElementText("ManagedEC2DeployDescription", Strings.ManagedEC2DeployDescription);
+            l.SetElementText("ManagedEC2DeployStatusLabel", Strings.ManagedEC2DeployStatusLabel);
+            l.SetElementText("ManagedEC2DeployStatusIcon", Strings.ManagedEC2DeployStatusIcon);
+            l.SetElementText("ManagedEC2DeployStatusText", Strings.ManagedEC2DeployStatusText);
+            l.SetElementText("ManagedEC2DeployActionsLabel", Strings.ManagedEC2DeployActionsLabel);
+            l.SetElementText("ManagedEC2CreateStackButton", Strings.ManagedEC2CreateStackButton);
+            l.SetElementText("ManagedEC2RedeployStackButton", Strings.ManagedEC2RedeployStackButton);
+            l.SetElementText("ManagedEC2DeleteStackButton", Strings.ManagedEC2DeleteStackButton);
+            l.SetElementText("ManagedEC2LaunchClientTitle", Strings.ManagedEC2LaunchClientTitle);
+            l.SetElementText("ManagedEC2LaunchClientLabel", Strings.ManagedEC2LaunchClientLabel);
+            l.SetElementText("ManagedEC2LaunchClientButton", Strings.ManagedEC2LaunchClientButton);
+        }
+
+        private string GetScenarioType(ElementLocalizer l) => _model.ScenarioIndex switch
+        {
+            1 => l.GetText(Strings.ManagedEC2ScenarioSingleFleetLabel),
+            3 => l.GetText(Strings.ManagedEC2ScenarioSpotFleetLabel),
+            4 => l.GetText(Strings.ManagedEC2ScenarioFlexFleetLabel),
+            _ => throw new ArgumentOutOfRangeException()
+        };
```

Comment:
This should be a mapping b/w the enum and the corresponding String

Reply:
Updated